src/frame/sdk/GroupApi.ts

```diff
@@ -114,6 +119,44 @@ function getCompleteListOfUserPublicKeys(userList: string[]) {
     });
 }
 
+/**
+ * Retrieves the master public keys for users in the memberList, adminList and owner. This will return new members list and admin list
+ * with the owner (if specified) added to the adminlist if it was not already on it.
+ * @param memberList
+ * @param adminList
+ */
```

Comment:
This should get updated to include the `owner` param

Reply:
changed